website/docs/cloud-docs/workspaces/index.mdx

```diff
@@ -51,32 +55,6 @@ Both HCP Terraform and Terraform CLI have features called workspaces, but they f
 
 - Terraform CLI workspaces are associated with a specific working directory and isolate multiple state files in the same working directory, letting you manage multiple groups of resources with a single configuration. The Terraform CLI does not require you to create CLI workspaces. Refer to [Workspaces](/terraform/language/state/workspaces) in the Terraform Language documentation for more details.
 
-## Listing and Filtering Workspaces
```

Comment:
Did you double check for this header name around to make sure no other pages link to it?